source/app/metrics/index.mjs

```diff
@@ -89,14 +89,6 @@
             else
               console.debug(`metrics/compute/${login} > optimize > this feature is currently disabled due to display issues (use --optimize flag in experimental features to force enable it)`)
           }
-        //Verify svg
-          if (verify) {
-            console.debug(`metrics/compute/${login} > verify SVG`)
-            const libxmljs = (await import("libxmljs")).default
-            const parsed = libxmljs.parseXml(rendered)
-            if (parsed.errors.length)
-              throw new Error(`Malformed SVG : \n${parsed.errors.join("\n")}`)
-          }
```

Comment:
This is required to check that units tests render correctly, it should be kept

Reply:
See [this](https://github.com/lowlighter/metrics/pull/186#discussion_r593754708)